kindelia/src/cli.rs

```diff
@@ -3,6 +3,7 @@ use std::path::PathBuf;
 
 use clap::{Parser, Subcommand};
 use clap_complete::Shell;
+use clap_num::maybe_hex;
```

Comment:
This is nice :v

Reply:
yeah, I was happy to find it.    nicer if it were part of core Clap and didn't require extra dep though.